tests/test_tags.py

```diff
@@ -217,25 +217,27 @@ def test_get_list_of_tags_with_host_filters_via_db(db_create_multiple_hosts, api
 
     with patch("api.tag.get_flag_value", return_value=True):
         for query in (
```

Comment:
The rest of the test would be cleaner if this was 2 vars instead of a tuple imo
```suggestion
        for query, expected_count in (
```

Reply:
Updated the test case. Not sure if this is what you had in mind.